static/app/views/replays/tabs.tsx

```diff
@@ -0,0 +1,34 @@
+import {TabList, Tabs} from 'sentry/components/tabs';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+
+const TABS = [
+  {key: 'replays', label: 'Replays', to: ''},
+  {key: 'selectors', label: 'Selectors', to: 'dead-rage-clicks'},
+];
+
+export default function ReplayTabs() {
+  const organization = useOrganization();
+  const hasDeadClickFeature = organization.features.includes(
+    'session-replay-rage-dead-selectors'
+  );
+
+  const location = useLocation();
+  const {pathname} = location;
+  const isSelectorIndex = pathname.includes('dead-rage-clicks');
```

Comment:
This is probably prone to breaking (e.g. if we change route path), but I can't think of a good solution right now. I think at the very least we can extract to a file level constant so that it is in sync with L#7

Reply:
i made them constants but wasn't sure what you meant by L#7!

Reply:
oh, we could add some props to this component, and have the value passed in. 

```
interface Props {
  selected: 'replays' | 'selectors';
}

export default function ReplayTabs({selected}: Props) {
  ...
```

Reply:
Oops sorry, L#7 = line #7